modules/atomic-widgets/props-resolver/settings-transformers/image-transformer.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\PropsResolver\SettingsTransformers;
+
+use Elementor\Modules\AtomicWidgets\PropsResolver\Transformer_Base;
+use Elementor\Modules\AtomicWidgets\PropTypes\Image_Prop_Type;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Image_Transformer extends Transformer_Base {
+	public function get_type(): string {
+		return Image_Prop_Type::get_key();
+	}
+
+	public function transform( $value ) {
+		if ( isset( $value['src']['id'] ) ) {
+			$image_src = wp_get_attachment_image_src(
+				(int) $value['src']['id'],
+				$value['size'] ?? 'full'
+			);
+
+			if ( ! $image_src ) {
+				throw new \Exception( 'Cannot get image src.' );
+			}
+
+			[ $src, $width, $height ] = $image_src;
+
+			return [
+				'src' => esc_url( $src ),
```

Comment:
remove all esc from tranformers
add `esc_url` in the `render` method of `atomic-image`

Reply:
Done